distro/vertx/src/main/resources/overlay/log4j2.xml

```diff
@@ -9,7 +9,7 @@ See more detail at: https://logging.apache.org/log4j/2.x/manual/async.html
 <Configuration status="WARN">
   <Appenders>
     <!-- Async Loggers will auto-flush in batches, so switch off immediateFlush. -->
-    <RandomAccessFile name="RandomAccessFile" fileName="apiman-gateway.log" immediateFlush="false" append="false">
+    <RandomAccessFile name="RandomAccessFile" fileName="apiman-gateway.log" immediateFlush="false" true="false">
```

Comment:
??

Reply:
I think you want this to be true (default)? Otherwise, we would overwrite some things.

> When true - the default, records will be appended to the end of the file. When set to false, the file will be cleared before new records are written.

Reply:
Yes, that's what my ?? comment to myself was about 😂.

Reply:
And indeed "true" or default is appropriate or it'll keep chopping the file after each flush.